moshi/src/main/java/com/squareup/moshi/AdapterMethodsFactory.java

```diff
@@ -87,6 +87,10 @@ public AdapterMethodsFactory(List<AdapterMethod> toAdapters, List<AdapterMethod>
           }
         }
       }
+
+      @Override public String toString() {
+        return "JsonAdapter" + annotations + "(" + type + ")";
```

Comment:
we use `type + " annotated " + annotations` in other places like exception messages